src/sql/workbench/contrib/dashboard/browser/contents/dashboardWidgetWrapper.component.ts

```diff
@@ -243,48 +243,52 @@ export class DashboardWidgetWrapper extends AngularDisposable implements OnInit
 	private updateTheme(theme: IColorTheme): void {
 		const el = <HTMLElement>this._ref.nativeElement;
 		const headerEl: HTMLElement = this.header.nativeElement;
-		let borderColor = theme.getColor(themeColors.DASHBOARD_BORDER);
-		let backgroundColor = theme.getColor(colors.editorBackground, true);
-		const foregroundColor = theme.getColor(themeColors.SIDE_BAR_FOREGROUND, true);
-		const border = theme.getColor(colors.contrastBorder, true);
+		if (this._config.showTitle === false) {
```

Comment:
the diff tool of github is pretty poor at handling this file, i am only adding a condition here and added indention for the rest of code since they are in else block now.